pyxem/tests/test_components/test_scattering_fit_component.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import numpy as np
+from pyxem.signals.reduced_intensity1d import ReducedIntensity1D
+from pyxem.components.scattering_fit_component import ScatteringFitComponent
+
+
+def test_scattering_component_init_lobato():
+    elements = ['Cu']
```

Comment:
condense this down to a one liner, `ScatteringFitComponent(['Cu'], [1], N=1., C=0., type='lobato')`

Reply:
Changed.